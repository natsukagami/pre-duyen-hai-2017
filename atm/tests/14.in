998 413
-5778746455743793920674283298198584436367997110830274682286875380980455099504948288750471902908879718507204544529835233255519983795218376546592481782765889254780876327644593945208269338363591440226587772085300702951093824939200147701103428644350152700992385732446578712213337626550933770770725546628647454826000604270597916816914026614751148071415273080279969934295920506218135440105154214525491814607087779090452457648997469348941415668503928027694683231915697667537782129792387969136814268353172167297335343160985286471600117214206502182099862620369586146874248097073962187074012877592554364884198966937380229914748092672382267318285117990151946126483190293894853278452927042588141959307733135770851829407548355502502597139888697517958388832974473658338413304296822162921161806937981723241327952251336067301581549155117690178531705355890184505152729257370883012948340077068478824194760863482652577832273260154584771937764553182085796478950948132152830778278527722120300645092238584101823217030682
