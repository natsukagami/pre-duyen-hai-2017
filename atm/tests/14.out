999999999999999999999999999999999999999999999999997757782129792387969136814268353172167297335343160985286471600117214206502182099862620369586146874248097073962187074012877592554364884198966937380229914748092672382267318285117990151946126483190293894853278452927042588141959307733135770851829407548355502502597139888697517958388832974473658338413304296822162921161806937981723241327952251336067301581549155117690178531705355890184505152729257370883012948340077068478824194760863482652577832273260154584771937764553182085796478950948132152830778278527722120300645092238584101823217030682
